modules/addStaticOnLocalStorage.js

```diff
@@ -0,0 +1,117 @@
+import { usersSettings } from './usersSettings.js'
+
+// Добавляет статистику по загрузкам в localStorage
+function addStaticOnLocalStorage() {
+    let elem = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload")
+
+    let promise = new Promise((resolve, reject) => {
```

Comment:
if you don't use the parameter `reject` , then you can add underscore before in order to avoid eslint error/warn .
And if you do not change the variable in code use const instead